beets/autotag/__init__.py

```diff
@@ -176,6 +177,7 @@ def apply_metadata(album_info: AlbumInfo, mapping: Mapping[Item, TrackInfo]):
             item.mb_artistid = album_info.artist_id
         item.mb_albumartistid = album_info.artist_id
         item.mb_releasegroupid = album_info.releasegroup_id
+        item.release_group_title = album_info.release_group_title
```

Comment:
Regarding the above: I think we can just delete this line, and the `for field, value in album_info.items():` loop below should have the same effect.